ui/src/app/base/sagas/http.js

```diff
@@ -47,9 +59,29 @@ export function* fetchScriptsSaga() {
       payload: response
     });
   } catch (error) {
+    // fetch doesn't return a complex error object, so we coerce the status text.
     yield put({
       type: `FETCH_SCRIPTS_ERROR`,
-      error: error.message
+      errors: { error: error.message }
+    });
+  }
+}
+
+export function* uploadScriptSaga(action) {
+  const csrftoken = yield call(getCookie, "csrftoken");
+  const script = action.payload;
+  let response;
+  try {
+    yield put({ type: `UPLOAD_SCRIPT_START` });
```

Comment:
There's a heap of places in these files that use the `` form of strings, I'm surprised prettier doesn't change that.